mlir/tools/mlir-tblgen/FormatGen.cpp

```diff
@@ -13,6 +13,8 @@
 
 using namespace mlir;
 using namespace mlir::tblgen;
+using llvm::SourceMgr;
+namespace cl = llvm::cl;
```

Comment:
We generally don't add usings for llvm::cl, I'd worry this is going to add inconsistencies in the tools.

Reply:
Done, I reverted the cl part of the change